src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -7,358 +7,694 @@ description: >
-called instances.
-
-For example, the variable font [Amstelvar](https://github.com/TypeNetwork/Amstelvar)
-has three masters for its **Weight** axis: The Regular master is at the center,
-and two masters, Thinner and Bolder, are at the opposite ends of the axis.
-Between these there are potentially 200 instances that can be chosen by the
-designer or developer:
+The font developer can offer a set of different axes, and because they all share
+the same default styles, you can combine them. Roboto has three styles in a
+Width axis: the Regular is at the center of the axis, and two styles, narrower
+and wider, are at each end. These provide all the widths of the Regular style,
+and combine with the Weight axis to provide all the widths for every weight.
 
 <figure class="w-figure">
-  <img src="Amstelvar-Alpha-example.png"
-  alt="Example of the Weight Axes for the typeface Amstelvar">
-  <figcaption class="w-figcaption">
-    Typeface Amstelvar, designed by David Berlow, type designer and
-    typographer at Font Bureau.
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="roboto-dance.mp4" type="video/mp4">
+  </video>
+  <figcaption>
+    Roboto Flex in random combinations of Width and Weight
   </figcaption>
 </figure>
 
-The [OpenType specification](https://docs.microsoft.com/en-gb/typography/opentype/spec/dvaraxisreg)
-specifies other axis, such as **Width**, **Optical Size**, **Italic** and
-**Slant**. All of these axes share the same default master, and we can combine
-them to explore an exponential number of typographic styles, like powers of 2,
-3, and 4 do to numbers.
-
-Amstelvar also has three masters in a Width axis: Again the Regular is at the
-center of the axis, and two masters, narrow and wider, are at the opposite ends
-of this axis. These not only provide all the widths of the Regular style, but
-also all the widths and weights combined.
-
-Among Amstelvar's registered axes (Width, Weight and Optical Size) there are
-thousands of styles. This may seem like massive overkill, but consider that
-Amstelvar only supports the Latin writing system. Considering the needs of all
-the world's scripts, and many of today's typography applications, the quality
-of the reading experience can be remarkably enhanced by this diversity of type
-styles within a font. And, if it is without performance penalty, the user can
-use a few or as many styles as they wish–it's up to their design.
-
-### Italics are slightly different
-
-The way that italics are handled in variable fonts is interesting, as there are
-two different approaches. Typefaces like Helvetica or Roboto have 
-interpolation-compatible contours, so their Roman and Italic styles can be interpolated
+This means there are thousands of styles! This may seem like massive overkill,
+but the quality of the reading experience can be remarkably enhanced by this
+diversity of type styles. And, if it is without performance penalty, web
```

Comment:
My question right now as your typical web developer is: what are these performance penalties and how do I watch out for them?